custom/skills.yaml

```diff
@@ -44,8 +44,7 @@ skills:
                 - [0.45, -2.3, -0.22, -1.4, 0.0]
             point_to_kinect:
                 - [0.01, 0.0, -1.57, -1.57, 0.0]
-        offset:
-            left:  {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            right: {x: 0.0, y: 0.0,  z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
+        left:
```

Comment:
Maybe call it just arm or center.

Reply:
That is not gonna work, since with this set-up the parameter is called using a.o. self.side, which is always left or right. I agree with you that this is not desired, but I do not think that that should be fixed in the pull request.

Reply:
But side could be center? This should be allowed in arms.py

Reply:
This breaks some code down the line unfortunately. A new issue has been made that treats this (#740), but for now I think the action_server pull request has priority. I assigned this issue to myself so I won't forget to do this and hopefully the arms can be made more generic using @alberth his stuff at that point as well